plasmapy/formulary/relativity.py

```diff
@@ -155,3 +155,25 @@ def relativistic_energy(m: u.kg, v: u.m / u.s) -> u.Joule:
     γ = Lorentz_factor(v)
     E = γ * m * c ** 2
     return E
+
+
+@validate_quantities(value={"can_be_negative": False})
+def relativistic_speed(
+    value: [u.J, u.kg * u.m / u.s],
+    particle: ParticleLike,
+    is_kinetic_energy: bool = False,
+) -> u.m / u.s:
```

Comment:
Add a docstring. Look at the other formulary functions to see how to format it, and what kind of information should be included. Make sure to explicitly explain the three separate ways this can be used 

1) Momentum -> velocity
2) KE -> velocity
3) Total energy -> velocity